interface/patient_file/summary/stats.php

```diff
@@ -31,452 +115,261 @@ function load_location(location) {
+        if (in_array($key, ["allergy", "medication"]) && $GLOBALS["erx_enable"]) {
+            $viewArgs['btnLabel'] = "Add";
+            $btnLinkPage = "eRx.php?page=medentry";
+        } else {
+            $viewArgs['btnLabel'] = "Edit";
+            $btnLinkPage = "patient_file/summary/stats_full.php?active=all&category=" . attr_url($key);
         }
+        $viewArgs['btnLink'] = str_replace("__page__", $btnLinkPage, $btnLinkBase);
 
-        while ($row = sqlFetchArray($pres)) {
-            // output each issue for the $ISSUE_TYPE
-            if (!$row['enddate'] && !$row['returndate']) {
-                $rowclass = "noend_noreturn";
-            } elseif (!$row['enddate'] && $row['returndate']) {
-                $rowclass = "noend";
-            } elseif ($row['enddate'] && !$row['returndate']) {
-                $rowclass = "noreturn";
-            }
-
-            echo " <tr class='text $rowclass;'>\n";
-
-            //turn allergies red and bold and show the reaction (if exist)
-            if ($key == "allergy") {
-                $reaction = "";
-                if (!empty($row['reaction'])) {
-                    $reaction = " (" . getListItemTitle("reaction", $row['reaction']) . ")";
-                }
-
-                echo "  <td colspan='" . attr($numcols) . "' style='color:red;font-weight:bold;'>&nbsp;&nbsp;" . text($row['title'] . $reaction) . "</td>\n";
-            } else {
-                echo "  <td colspan='" . attr($numcols) . "'>&nbsp;&nbsp;" . text($row['title']) . "</td>\n";
-            }
-
-            echo " </tr>\n";
+        if (count($listData) == 0) {
+            $viewArgs['listTouched'] = (getListTouch($pid, $key)) ? true : false;
         }
 
-        echo "</table>";
-        if ($_POST['embeddedScreen']) {
-            echo "</div></td></tr>";
-        }
+        echo $t->render('patient/card/medical_problems.html.twig', $viewArgs);
     }
 }
-?>
-</table> <!-- end patient_stats_issues -->
 
-<table id="patient_stats_spreadsheets">
-<?php
-
-// Show spreadsheet forms if any are present.
-//
-$need_head = true;
-foreach (array('treatment_protocols','injury_log') as $formname) {
-    if (sqlNumRows(sqlStatement("SHOW TABLES LIKE ?", array("form_" . $formname))) > 0) {
-        $dres = sqlStatement("SELECT tp.id, tp.value FROM forms, " .
-                            "form_" . add_escape_custom($formname) .
-                " AS tp WHERE forms.pid = ? AND " .
-                            "forms.formdir = ? AND tp.id = forms.form_id AND " .
-                            "tp.rownbr = -1 AND tp.colnbr = -1 AND tp.value LIKE '0%' " .
-                            "ORDER BY tp.value DESC", array($pid, $formname));
+// Render Cards for 2 specific forms
+foreach (['treatment_protocols', 'injury_log'] as $formname) {
+    if (sqlNumRows(sqlStatement("SHOW TABLES LIKE ?", ["form_{$formname}"])) > 0) {
+        $sql = "SELECT tp.id, tp.value
+            FROM forms, ? AS tp
```

Comment:
this should break (can't bind table names or columns). would just make it:
```
FROM forms, form_{$formname} AS tp
```
(and drop it from the binding of the variable below)